beetsplug/chroma.py

```diff
@@ -57,6 +59,29 @@ def prefix(it, count):
         yield v
 
 
+def releases_key(release, countries, original_year):
+    """Used as a key to sort releases by date then preferred country
+    """
+    date = release.get('date')
+    if date and original_year:
+        year = date.get('year', 9999)
+        month = date.get('month', 99)
+        day = date.get('day', 99)
+        key = '{0:04d}{1:02d}{2:02d}'.format(year, month, day)
```

Comment:
Any particular reason this produces a string instead of a tuple? Python will happily sort tuples indexwise:

```py
>>> sorted([(1,2,3), (1,1,1)])
[(1, 1, 1), (1, 2, 3)]
```